src/components/Divider/Divider.js

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import styles from './style.scss';
+
+const Divider = () => <div className={styles.divider} />;
```

Comment:
Shouldn't we use `restProps` here? For instance, there is no way to change `width` of this component. Devs will need wrapper div to handle it.

Reply:
ada5c0d4ab1a71b4d2fe5edbf16b8421399e55dd

Reply:
can u use `getMergedClassNames` or cx here to unlock passing className without overriding our styles?

Reply:
done at f9f60b89b7e70ef2936e66f6786773861f232de9